src/commands/init.js

```diff
@@ -80,6 +83,96 @@ async function handleDatabaseConfiguration() {
   return inquirer.prompt(promptContent);
 }
 
+function validateEndpoint(input) {
+  if (!VALIDATION_REGEX_URL.test(input)) {
+    return 'Application input must be a valid url.';
+  }
+  if (!VALIDATION_REGEX_HTTPS.test(input)) {
+    return 'HTTPS protocol is mandatory, except for localhost and 127.0.0.1.';
+  }
+  return true;
+}
+
+function getApplicationPortFromCompleteEndpoint(endpoint) {
+  if (endpoint.startsWith('http://localhost:') || endpoint.startsWith('http://127.0.0.1:')) {
+    return endpoint.substring(17);
+  }
+  return null;
+}
```

Comment:
[Misc] Does that mean that we choose not to handle any other local endpoints?

Reply:
What other local endpoint could exist?
Maybe `::1`? 🤔

Reply:
I'll correct it to match all endpoint ports.